.ci/releasenotes-gen-auto.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
+
+echo "PULL_REQUEST:""$PULL_REQUEST"
+if [[ $PULL_REQUEST =~ ^([0-9]+)$ ]]; then
+  echo "Build is not for Pull Request";
+  sleep 5;
```

Comment:
please remove.

`sleep 5` was required in Travis jobs in some time in past, as they had defect in persistence of logs. 
We migrated off Travis and changed in Travis architecture of execution, such problem is not reproducing.
We do not update old scripts, but in new script we try to void this.

Reply:
Changes Done